crates/rpc/rpc-eth-api/src/helpers/fee.rs

```diff
@@ -87,6 +98,9 @@ pub trait EthFees: LoadFee {
                 .map_err(Self::Error::from_eth_err)?
                 .ok_or(EthApiError::HeaderNotFound(newest_block.into()))?;
 
+            let pending_block =
+                if newest_block.is_pending() { self.local_pending_block().await? } else { None };
```

Comment:
looks like it is unreachable because of 
https://github.com/paradigmxyz/reth/blob/09506aa130b4c0dae3fb7029f27845b4f2c58321/crates/rpc/rpc-eth-api/src/helpers/fee.rs#L78-L84

Reply:
yep